flocker/provision/_libcloud.py

```diff
@@ -197,153 +195,6 @@ def get_ssh_key(self):
             # https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_KeyPairInfo.html
             return None
 
-    def create_node(self, name, distribution, metadata={}):
```

Comment:
I *think* the logic here (and for AWS) is mostly the same for both the single node and multiple node code paths, but it is a bit hard to follow the code to be sure. The one difference that I am aware of is that the single-node libcloud case retries on failure, but this is handled at higher level in the multi-node case (in `admin.acceptance`).